docs/pages/choose-an-edition/teleport-enterprise/hsm.mdx

```diff
@@ -64,41 +64,67 @@ On the EC2 instance where you will run your Teleport Auth Server:
   <TabItem label="YubiHSM2">
 1. Install the YubiHSM2 [SDK](https://developers.yubico.com/YubiHSM2/Releases/).
 
-2. Start `yubihsm-connector`
+   <Admonition type="warning">
+     The YubiHSM2 SDK version `2023.01` is currently unsupported.
+     The latest known supported version is `2022.06`.
+   </Admonition>
+
+2. Start `yubihsm-connector` with debug logging enabled. This is a background
+   process that you will need to keep running to facilitate connections to your
+   YubiHSM2.
 
    ```code
-   $ yubihsm-connector
+   $ yubihsm-connector -d
+   DEBU[0000] preflight complete                            cert= config= key= pid=73502 seccomp=false serial= syslog=false timeout=0s version=3.0.3
+   DEBU[0000] takeoff                                       TLS=false listen="localhost:12345" pid=73502
    ```
-3. Use `yubihsm-shell` to create a new authentication key to be used by
-   Teleport with the below capabilities.
-   <Admonition type="note">
-   YubiHSM2 comes with a default authentication key at slot 1 with password
+3. Use `yubihsm-shell` to create a new Authentication Key to be used by
+   Teleport with the necessary capabilities.
+
+   YubiHSM2 comes with a factory default Authentication Key at slot 1 with password
    `password`. You should replace and delete it as recommended by Yubico.
 
-   When creating the authentication key to be used by Teleport, the password
+   When creating the Authentication Key to be used by Teleport, the password
    must have at least 8 characters. The example `hunter22` is used here.
-   </Admonition>
 
    ```text
```

Comment:
With the additional output here, I suggest using `code` here. If you don't like the muted colors of the output in this form, you can separate the output into their own blocks.

Reply:
I'm not using `code` because the `yubihsm-shell` has the custom prompt `yubihsm>` which is not recognized by the `code` block formatter. The line `yubihsm> connect` is not output, the user is expected to run the `connect` command.

I could update this block to use the generic `$` prompt, I just thought the `yubihsm>` prompt made it extremely clear that these commands should be run in the `yubihsm-shell` program, and this is closer to what the user will actually see. What do you think?